openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -38,7 +38,7 @@ OpenAssessment.BaseView.prototype = {
      */
     scrollToTop: function() {
         if ($.scrollTo instanceof Function) {
-            $(window).scrollTo($("#openassessment__steps"), 800, {offset:-50});
+            $(window).scrollTo($(this.element).find("#openassessment__steps"), 800, {offset:-50});
```

Comment:
@La0 minor nit: jQuery allows specifying a [`context` parameter](http://api.jquery.com/jQuery/) to `$`, so this is equivalent to:

    $(window).scrollTo($("#openassessment__steps", this.element), 800, {offset:-50});

which is a bit cleaner.